apps/client/src/common/hooks/useEventAction.ts

```diff
@@ -152,6 +153,54 @@ export const useEventAction = () => {
     [_updateEventMutation],
   );
 
+  /**
+   * Calls mutation to edit multiple events
+   * @private
+   */
+  const _batchUpdateEventsMutation = useMutation({
+    mutationFn: requestBatchPutEvents,
+    onMutate: async ({ ids, data }) => {
+      // cancel ongoing queries
+      await queryClient.cancelQueries([RUNDOWN_TABLE_KEY]);
```

Comment:
we have RUNDOWN_TABLE, which would be the same

Reply:
Not nitpicky at all. Changed